nav2_graceful_controller/include/nav2_graceful_controller/graceful_controller.hpp

```diff
@@ -163,6 +169,7 @@ class GracefulController : public nav2_core::Controller
   Parameters * params_;
   double goal_dist_tolerance_;
   bool goal_reached_;
+  bool has_new_path_;
```

Comment:
I think maybe this just needs to be renamed - it's not so much that "we have a new path" - but rather "we are still doing the initial rotation"

Reply:
OK! I don't feel overly strongly about this - just felt like a reasonable separation of concerns

Reply:
I think you're actually right that the name is bad though - because it doesn't JUST indicate a new path has been set (you can see we set this flag when a new path arrives, but we don't clear it until we have successfully done the whole initial rotation)